### ISSUE TITLE
Cannont disable parental controls if NNID is for a child

### ISSUE THREAD

**Issue by Kingwad** [Plailect/Guide#962]
If the linked NNID is for a child under 13, you cannot fully clear parental controls and remove the pin, making the not "Disable the parental controls of your device before doing the 2.1.0 ctrtransfer. If you do not know the password, see this site." difficult to follow.

**Comment by Zomnut** [comment]
Anecdotally, I succeeded with the following steps (single test case, no replication possible for me)

1. Took a NANDmin backup
2. Removed all parental restrictions individually (parental controls are not cleared, pin is still set, but all options are set to "do not restrict")
3. Opened the web browser to make sure I wasn't prompted for a pin
4. Did the ctrtransfer (skipping backup, as I just took one in step 1)

The result: success, and after restoring the backup, all parental controls were as they had been setup originally.

I'm not sure how much detail should be added to the "Disable the parental controls ..." step, but it may deserve a "troubleshooting" entry.
